commands/language.js

```diff
@@ -0,0 +1,40 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const {MessageActionRow, MessageEmbed, MessageSelectMenu, SelectMenuInteraction} = require('discord.js');
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('language')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/EvilInsultGenerator/discord-bot/pullRequest?prid=8755514)
```suggestion
        .setName("language")
```